ansible/roles/config/templates/config.toml.tmpl

```diff
@@ -96,6 +95,8 @@ imports = ["/etc/containerd/conf.d/*.toml"]
           runtime_engine = ""
           runtime_root = ""
           privileged_without_host_devices = false
+          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
+            systemd_cgroup = true
```

Comment:
Is this value dependent on the Kubernetes version?

Reply:
Yes, but as of v1.21, kubeadm `init` configures kubelet to use the systemd cgroup driver, and as of v1.22, _all_ kubeadm commands configure kubelet to use it.

Please see https://github.com/kubernetes/kubeadm/issues/2376#issuecomment-788252505